api.planx.uk/modules/analytics/metabase/shared/client.ts

```diff
@@ -0,0 +1,155 @@
+import axios from "axios";
+import type {
+  AxiosInstance,
+  AxiosError,
+  AxiosRequestConfig,
+  InternalAxiosRequestConfig,
+} from "axios";
+import dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
I think this is a legacy from the script days of this class - this is already handled in the API. You should be able to remove this and still access `process.env` values.